File: C_Workspace/C_CheckList/Arrays/1-d_Arrays/11.c

//C program to find occurrence of an element in one dimensional array.

#include <stdio.h>

int main() {
    int n, i, key, count = 0;

    printf("Enter number of elements: ");
    scanf("%d", &n);

    int arr[n];

    printf("Enter %d elements:\n", n);
    for (i = 0; i < n; i++)
        scanf("%d", &arr[i]);

    printf("Enter element to count: ");
    scanf("%d", &key);

    for (i = 0; i < n; i++) {
        if (arr[i] == key)
            count++;
    }

    printf("%d occurs %d times\n", key, count);

    return 0;
}

File: Python_Workspace/Python_Tasks/15_06/11.py
#Palindrome number
a=int(input("Enter the number:"))
b=a
x=0
while a>0:
  r=a%10
  x=x*10+r
  a=a//10
if(x==b):
  print(b,"=",x,"is a Palindrome number")
else:
  print(b,"!=",x,"is not a Palindrome number")
